testing.go

```diff
@@ -3,13 +3,19 @@ package plugin
 import (
 	"bytes"
 	"context"
+	"io"
 	"net"
 	"net/rpc"
 
 	"github.com/mitchellh/go-testing-interface"
 	"google.golang.org/grpc"
 )
 
+type TestOptions struct {
+	ServerStdout io.ReadCloser
+	ServerStderr io.ReadCloser
```

Comment:
It would be preferrable to have doc comments on these.

Reply:
Fair enough, will do that.